Makefile

```diff
@@ -85,12 +85,12 @@ GITHUB_REF_NAME ?= $(shell git rev-parse --abbrev-ref HEAD)
 # backwards compatible to build with Drone
 ifneq ($(DRONE_TAG),)
 	GITHUB_REF_TYPE := tag
-	GITHUB_REF_NAME := DRONE_TAG
+	GITHUB_REF_NAME := $(DRONE_TAG)
 endif
 
 ifneq ($(GITHUB_REF_TYPE),branch)
 	VERSION ?= $(subst v,,$(GITHUB_REF_NAME))
-	GITEA_VERSION ?= $(GITHUB_REF_NAME)
+	GITEA_VERSION ?= $(VERSION)
```

Comment:
This changes the behaviour of `GITEA_VERSION` as `VERSION` is only supposed to have the `v` trimmed, but `GITEA_VERSION` doesn't.

Reply:
Isn't that how it used to be?
https://github.com/go-gitea/gitea/blob/e24f651c86a5168da8181f2c75f5f99b56561224/Makefile#L83-L84

Reply:
Ah, you're right. I was looking at  the wrong diff.